### ISSUE TITLE
Pause the great suspender

### ISSUE THREAD

**Issue by Davidrl1000** [deanoemcke/thegreatsuspender#256]
There should be a way to pause the great suspender. There are some moments when the people (ie: I) need to work with all the tabs an is a little uncomfortable to be pausing all the tabs

**Comment by resmall** [comment]
:+1:

**Comment by deanoemcke** [comment]
The workaround for this is to change your suspend time in options to 'never'. That will effectively pause the extension.

**Comment by deanoemcke** [comment]
I agree that this may be worth implementing if there is enough support for it.

**Comment by wdhwg001** [comment]
+1 for this. Any progress on this feature?

**Comment by euphoricdisstopia** [comment]
+1. Would really love the ability to pause the Great Suspender when I'm at work and have like 15+ tabs open that need to stay open. But once the workday is over, suspend suspend suspend those tabs.

**Comment by ossilator** [comment]
#818 suggests a more selective and automated approach - that seems more helpful than this manual "kill switch".
